assets/diagram-js.css

```diff
@@ -508,86 +508,305 @@ marker.djs-dragger tspan {
 /**
  * popup styles
  */
-.djs-popup .entry {
+ .djs-popup {
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 200;
+  line-height: 1;
+  font-family: "IBM Plex Sans", sans-serif;
+  background: var(--popup-background-color);
+  border: solid 1px var(--popup-border-color);
+  border-radius: 2px;
+
+  --color-white: #fff;
```

Comment:
Should we re-use colors defined globally here or move these colors to the global scope?

I'd love to move variable definitions to the global scope, too:

https://github.com/bpmn-io/diagram-js/blob/develop/assets/diagram-js.css#L5

+ prefix popup styles there with `--popup-`

Reply:
Got cha! Maybe we find a way to re-use these styles / variables regardless, i.e. introduce a `djs-parent` selector that provides them to different roots (popup-menu + djs-container)?